src/schema/submission.yaml

```diff
@@ -819,9 +819,9 @@ classes:
       - submission_date
       - submission_status
     slot_usage:
-      affiliations:
+      id:
         description: >-
-          Institution/Center/Data Hub that is providing this submission.
+          A internal unique identifier for the Submission.
```

Comment:
The schema only describes the content of a submission. The ID is not something the user chooses and can be seen as operational metadata for the submission. Thus I would rather remove it.

Same applies to submission_date and submission_status.